iterative/utils/logger.go

```diff
@@ -0,0 +1,104 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/sirupsen/logrus"
+	"strings"
+	"time"
+)
+
+var baseTimestamp = time.Now()
+var colors = make(map[string]int)
+
+type basicFormatter struct{}
+
+func (f *basicFormatter) Format(entry *logrus.Entry) ([]byte, error) {
+	levelText := strings.ToUpper(entry.Level.String())
+	levelColor := colors[levelText]
+	tpl := "[%s] 🚀\x1b[%dmTPI\x1b[0m %s\n"
+	return []byte(fmt.Sprintf(tpl, levelText, levelColor, entry.Message)), nil
+}
+
+func init() {
+	colors["DEBUG"] = 36
+	colors["INFO"] = 36
+	colors["WARN"] = 33
+	colors["ERROR"] = 31
+	colors["FATAL"] = 31
+	colors["purple"] = 35
```

Comment:
purple? why?

If it's about iterative colours should we add cyan & orange too?